Bloon/Features/SBA Inactivity/SBAActivityTrackingFeature.cs

```diff
@@ -0,0 +1,78 @@
+namespace Bloon.Features.SBAInactivity
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Bloon.Core.Database;
+    using Bloon.Core.Discord;
+    using Bloon.Core.Services;
+    using Bloon.Variables.Channels;
+    using DSharpPlus;
+    using DSharpPlus.EventArgs;
+    using Microsoft.Extensions.DependencyInjection;
+
+    public class SBAActivityTrackingFeature : Feature
+    {
+        private readonly IServiceScopeFactory scopeFactory;
+        private readonly DiscordClient dClient;
+
+        public SBAActivityTrackingFeature(IServiceScopeFactory scopeFactory, DiscordClient dClient)
+        {
+            this.scopeFactory = scopeFactory;
+            this.dClient = dClient;
+        }
+
+        public override string Name => "SBA Activity Tracking";
+
+        public override string Description => "Tracks message history of users within SBA.";
+
+        public override Task Initialize()
+        {
+            this.dClient.MessageCreated += this.TrackSBAAsync;
```

Comment:
As this is now a separate feature, this should be moved to `OnEnable()` and respectively `OnDisable()`